rowcodec/common.go

```diff
@@ -13,6 +13,7 @@ import (
 // CodecVer is the constant number that represent the new row format.
 const CodecVer = 128
 
+var EnableShardKeySupport = false
```

Comment:
How about remove this variable and make two isRowKey functions, I think that would be cleanner, maintains the state in a single place.